emperor/support_files/js/emperor.js

```diff
@@ -879,23 +879,26 @@ function toggleLabels() {
 
 /*This functions handles the display of taxonomic labels*/
 function displayTaxaLabels(taxaLevel){
-        var labelshtml = "";
-        var taxonomy = ["Full","Kingdom","Phylum","Class","Order","Family","Genus","Species"];
-
-        $('#taxalabels').css('display','block');
-        $("#taxaLevel" ).html(taxonomy[taxaLevel]);
-        for(var key in g_taxaPositions){
-                // get the coordinate of this taxa sphere
-            	var coords = toScreenXY(g_plotTaxa[key].position,g_sceneCamera,$('#main-plot'));
-
-            	// labels are identified by the key they have in g_taxaPositions
-            	labelshtml += "<label id=\""+key+"_taxalabel\" class=\"unselectable labels\" style=\"position:absolute; left:"+parseInt(coords['x'])+"px; top:"+parseInt(coords['y'])+"px;\">";
-               labelshtml += truncateLevel(g_taxaPositions[key]['lineage'], taxaLevel);
-            	labelshtml += "</label>";
+        if(!jQuery.isEmptyObject()){
```

Comment:
the 2 blocks within the ifs are over indented and the first line of the second block has an extra space.